src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -759,6 +765,55 @@ CHIP_ERROR P256Keypair::NewCertificateSigningRequest(uint8_t * out_csr, size_t &
     return error;
 }
 
+CHIP_ERROR VerifyCertificateSigningRequest(const uint8_t * csr_buf, size_t csr_length, P256PublicKey & pubkey)
+{
+#if !CHIP_TARGET_STYLE_EMBEDDED
```

Comment:
should we have a different constant name here?

I am thinking that a name of 'CHIP_TARGET_SUPPORTS_DER_PARSING' or 'CHIP_TARGET_SUPPORT_CERTIFICATE_SIGNING' may be a clearer name even if they are aliased to TARGET_STYLE_EMBEDDED.